server/controllers/stock/index.js

```diff
@@ -336,21 +337,76 @@ function listLotsOrigins(req, res, next) {
  * create a new integration entry
  */
 function createIntegration(req, res, next) {
+  const transaction = db.transaction();
   const params = req.body;
   const identifier = uuid.v4();
+  const documentUuid = uuid.v4();
+  let commonInfos;
+
   const integration = {
     uuid        : db.bid(identifier),
     project_id  : req.session.project.id,
-    description : params.description || 'INTEGRATION',
-    date        : new Date(),
+    description : params.integration.description || 'INTEGRATION',
```

Comment:
Hmm.. it seems like we should either make this required or do this on the client.  It seems like magic that this word is written into the database.  I know you didn't write this code originally, so we should just schedule it for a later refactor.  Maybe when we introduce vouchers.

Reply:
I agree, I propose to tackle it in voucher introduction